srtcore/core.cpp

```diff
@@ -1050,6 +1050,17 @@ void CUDT::getOpt(SRT_SOCKOPT optName, void *optval, int &optlen)
         optlen             = sizeof(int64_t);
         break;
 
+    case SRTO_ESTINPUTBW:
+        *(int64_t*)optval = 0LL;
+        if (m_pSndBuffer && m_pSndBuffer->getInRatePeriod() != 0)
```

Comment:
`int64_t` to conform with `SRTO_MAXBW` and `SRTO_INPUTBW`, while internally it's type is just `int`.

Reply:
That's perfectly ok.